README.adoc

```diff
@@ -54,7 +54,9 @@ with each other in different containers.
 
 Before you begin, Podman needs to be installed. For installation instructions, refer to the https://podman.io/getting-started/installation[official Podman documentation^]. You will build and run the microservices in containers.
 
-If you are running Mac or Windows, make sure to start your Podman-managed VM before you proceed.
+If you are running Mac or Windows, make sure to initialize and start your Podman-managed VM before you proceed.
+
+On Windows, Podman runs in rootless mode by default, which does not assign a container IP. Because the `system` and `inventory` microservices need to communicate between containers, you must initialize Podman in https://docs.podman.io/en/latest/markdown/podman-machine-init.1.html#rootful[rootful^] mode to enable proper networking.
```

Comment:
```suggestion
On Windows, Podman runs in rootless mode by default, which does not assign a container IP address. Because the `system` and `inventory` microservices need to communicate between containers, you must initialize Podman in https://docs.podman.io/en/latest/markdown/podman-machine-init.1.html#rootful[rootful^] mode to enable proper networking.
```